### DISCUSSION TITLE
Custom withdraw and deposit values

### DISCUSSION THREAD

**Discussion by IromTypo** [runelite/runelite#15883] [Q&A] [answered]
Hi,

I withdraw 10k runes for my rune pouch, (so more can fit in my rune pouch :)) I want 40 barley seeds for my bird house run, and 9 herb seeds for my herb run. Constantly having to type in different numbers is hella annoying.

It would be super helpful if there was a plugin that would add a custom amount menu (like the shift click menu) with preset custom amounts.

Very, very much appreciated!

**Comment by Nightfirecat** [comment, marked as answer]
This isn't possible--the client doesn't support multiple custom X values for depositing/withdrawing.
